apps/test/unit/code-studio/components/header/EditableProjectNameTest.js

```diff
@@ -18,9 +18,9 @@ describe('EditableProjectName', () => {
     currentName = 'Brand New Project';
     replaceOnWindow('dashboard', {
       project: {
-        rename: (name, cb) => {
+        rename: name => {
           currentName = name;
-          cb();
+          return Promise.resolve();
```

Comment:
This had to be updated because the signature of `dashboard.project.rename` changed.